businfo/models/model.go

```diff
@@ -0,0 +1,46 @@
+package models
+
+type BusInfo struct {
+	Message interface{} `json:"message"`
+	Errors  interface{} `json:"errors"`
+	Data    []struct {
+		Id          string   `json:"id"`
+		Date        string   `json:"date"`
+		RouteId     string   `json:"routeId"`
+		CoachNo     string   `json:"coachNo"`
+		CoachType   string   `json:"coachType"`
+		SeatTypesId []string `json:"seatTypesId"`
+		Fares       struct {
+			EClass string `json:"E-Class,omitempty"`
+			ECLASS string `json:"E-CLASS,omitempty"`
+			BClass string `json:"B-Class,omitempty"`
+			BCLASS string `json:"B-CLASS,omitempty"`
+		} `json:"fares"`
+		RouteName      string  `json:"routeName"`
+		AvailableSeats int     `json:"availableSeats"`
+		StartCounter   string  `json:"startCounter"`
+		EndCounter     *string `json:"endCounter"`
+		ArrivaleTime   *string `json:"arrivaleTime"`
+		DepartureTime  string  `json:"departureTime"`
+		DiscountDetail *struct {
+			DiscountApplicable bool        `json:"discountApplicable"`
+			DiscountType       interface{} `json:"discountType"`
+			DiscountAmount     interface{} `json:"discountAmount"`
+			DiscountSeats      interface{} `json:"discountSeats"`
+		} `json:"discountDetail"`
+		CompanyName string `json:"companyName"`
+		Fare        string `json:"fare"`
+	} `json:"data"`
+}
+
+type NotificationPld struct {
```

Comment:
What is the purpose of this? Judging by the code, it's not named properly.

Reply:
It was initially intended for the User notification in tgbot. I will rethink.